api/MediaSource.json

```diff
@@ -103,38 +103,38 @@
             "standard_track": true,
             "deprecated": false
           }
-        }
-      },
-      "worker_support": {
-        "__compat": {
-          "description": "Available in workers",
-          "support": {
-            "chrome": {
-              "version_added": "108"
-            },
-            "chrome_android": "mirror",
-            "edge": "mirror",
-            "firefox": {
-              "version_added": false
-            },
-            "firefox_android": "mirror",
-            "ie": {
-              "version_added": false
-            },
-            "oculus": "mirror",
-            "opera": "mirror",
-            "opera_android": "mirror",
-            "safari": {
-              "version_added": false
+        },
+        "worker_support": {
+          "__compat": {
+            "description": "Available in workers",
+            "support": {
+              "chrome": {
+                "version_added": "108"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
             },
-            "safari_ios": "mirror",
-            "samsunginternet_android": "mirror",
-            "webview_android": "mirror"
-          },
-          "status": {
-            "experimental": true,
-            "standard_track": true,
-            "deprecated": false
+            "status": {
+              "experimental": true,
```

Comment:
I noticed that MSE-in-Workers in MDN in these changes is being marked "experimental". It is shipping on-by-default in Chrome, and is implemented to follow the corresponding changes in the current MSE spec working draft. What does "experimental" mean here, given this information?

Reply:
This is the definition that MDN uses for experimental: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#experimental. In the time I've been contributing to MDN, we've gone back and forth on this a lot, and it still isn't perfect, but I think it's OK. MSE-in-workers is marked as experimental because it is still only available in one rendering engine, so the advice to devs would be to think carefully when considering using it. If you want to support Fx and WebKit browsers, then you might need to think about a fallback, alternative, or gracefully degrading experience. If you only need Chromium support, then you are OK. This certainly isn't the same as "don't use it".

Reply:
Thanks for the explanation. https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#experimental could probably use clarification around the meaning of "It is implemented and enabled by default in less than two modern major browsers" -- I understand that it is being applied as if s/major browsers/major distinct underlying rendering engines in browsers/, but on the surface it is confusing.

Reply:
I've filed this: https://github.com/mdn/content/issues/22391. Please add to it if you have any other thoughts on the subject. I'll try to fix this at some point next week, if no one else grabs it.